client/src/components/J40Header.tsx

```diff
@@ -89,67 +93,50 @@ const J40Header = () => {
           className={'j40-header'}>{intl.formatMessage(messages.timeline)}</Link>],
     ]);
 
-    // select which items from the above map to show, right now it's only two
-    // possibilities so it's simple. Note: strings are used as react keys
-    const menu =
-      ('sprint3' in flags!) ?
-        ['about', 'cejst', 'methodology', 'contact'] :
-        ['about', 'cejst', 'methodology', 'contact'];
-    // TODO: make feature flags flags work.
+    const menu =['about', 'cejst', 'methodology', 'contact'];
     return menu.map((key) => menuData.get(key));
   };
 
   return (
     <>
-      <Helmet htmlAttributes={{lang: intl.locale}}>
-        <meta charSet="utf-8"/>
-        <title>{title}</title>
-      </Helmet>
       <GovBanner/>
       <Header
         basic={true} role={'banner'}
         className={'usa-header j40-header'}>
         <div className="usa-nav-container">
           <div className="usa-navbar">
-            <Title>{title}
-              <div className={'byline'}>
-                A climate and economic justice screening tool
-              </div>
-            </Title>
-
+            <h1 className="usa-logo">
+              <img className="j40-sitelogo" src={siteLogo} alt={`${titleL1} ${titleL2}`} />
+              <span className={'usa-logo__text j40-title'}>
+                <span className={'j40-title-line1'}>{titleL1}</span><br/>
+                <span className={'j40-title-line2'}>{titleL2}</span>
+              </span>
+            </h1>
             <NavMenuButton
               key={'mobileMenuButton'}
               onClick={toggleMobileNav}
               label="Menu"/>
           </div>
           <PrimaryNav
-            items={headerLinks(flags)}
+            items={headerLinks()}
             mobileExpanded={mobileNavOpen}
             onToggleMobileNav={toggleMobileNav}
             className={'j40-header'}
           >
           </PrimaryNav>
         </div>
       </Header>
-      <Alert
-        className={'j40-sitealert'}
-        type="info">
-        <b>Public beta — </b>
-        Welcome to the public beta of the Just Progress Map, a climate and
-        economic justice screening tool. The tool will be continuously updated.
-        Please submit feedback.
+      <Alert className={'j40-sitealert'} type="info">
+        <b><FormattedMessage
```

Comment:
bold of you to use a bold tag 😺

Reply:
Yeah we should probably use CSS here

Reply:
Yeah. Agreed. The title and message are now in spans with css classnames that can control them.